skills/declarative/whoSkill/whoSkill/schemas/sdk.uischema

```diff
@@ -562,6 +562,167 @@
       "widget": "ActionCard"
     }
   },
+  "Microsoft.Graph.Calendar.AcceptEvent": {
+    "menu": {
+      "submenu": [
+        "Microsoft Graph"
+      ]
+    }
+  },
+  "Microsoft.Graph.Calendar.CreateEvent": {
+    "menu": {
+      "submenu": [
+        "Microsoft Graph"
+      ]
+    }
+  },
+  "Microsoft.Graph.Calendar.DeclineEvent": {
+    "menu": {
+      "submenu": [
+        "Microsoft Graph"
+      ]
+    }
+  },
+  "Microsoft.Graph.Calendar.DeleteEvent": {
+    "menu": {
+      "submenu": [
+        "Microsoft Graph"
+      ]
+    }
+  },
+  "Microsoft.Graph.Calendar.FindMeetingTimes": {
+    "menu": {
+      "submenu": [
+        "Microsoft Graph"
+      ]
+    }
+  },
+  "Microsoft.Graph.Calendar.GetContacts": {
+    "menu": {
+      "submenu": [
+        "Microsoft Graph"
+      ]
+    }
+  },
+  "Microsoft.Graph.Calendar.GetEventById": {
+    "menu": {
+      "submenu": [
+        "Microsoft Graph"
+      ]
+    }
+  },
+  "Microsoft.Graph.Calendar.GetEvents": {
+    "menu": {
+      "submenu": [
+        "Microsoft Graph"
+      ]
+    }
+  },
+  "Microsoft.Graph.Calendar.GetWorkingHours": {
+    "menu": {
+      "submenu": [
+        "Microsoft Graph"
+      ]
+    }
+  },
+  "Microsoft.Graph.Calendar.Helpers.FilterEvents": {
+    "menu": {
+      "submenu": [
+        "Calendar Skill Helpers"
```

Comment:
Looks like this hasn't updated

Reply:
Good catch I forgot to push the merged sdk.uischema!